ArduCopter/defines.h

```diff
@@ -43,7 +43,7 @@ enum tuning_func {
     TUNING_ALTITUDE_HOLD_KP =           14, // altitude hold controller's P term (alt error to desired rate)
     TUNING_RATE_ROLL_PITCH_KD =         21, // body frame roll/pitch rate controller's D term
     TUNING_VEL_XY_KP =                  22, // loiter rate controller's P term (speed error to tilt angle)
-    TUNING_ACRO_RP_KP =                 25, // acro controller's P term.  converts pilot input to a desired roll, pitch or yaw rate
+    TUNING_ACRO_RP_RATE =               25, // acro controller's desired roll and pitch rate in deg/s
```

Comment:
this is slightly dangerous to re-use the old tuning value because the valid range is very different (I think).  I wonder if perhaps we should just remove the ability to tune this parameter at all.  Is it really necessary to be able to change the value with a tuning knob?  Perhaps it was more necessary previously when the values were impossible to understand.

Reply:
OK, Leonard wants to leave this ability in so it's ok.  It's a bit of an edge case to worry about this.

So we should update the TUNE parameter description.

Reply:
Done